src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -208,7 +157,42 @@ const DatasetSchemaInformationView = withStyles(styles)(({ classes }: IProps) =>
         />
       </Grid>
 
-      <Grid item xs={6} className={classes.formFieldContainer}>
+      <Grid item xs={6}>
+        <label
+          htmlFor="dataset-defaultProfileId"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.defaultProfile })}
+        >
+          Billing Profile*
+        </label>
+        <Controller
+          name="defaultProfileId"
+          control={control}
+          rules={{ required: 'default billing profile is required' }}
+          render={({ field }) => (
+            <Autocomplete
+              id="dataset-defaultProfileId"
+              options={profileOptions}
+              className={classes.formInput}
+              isOptionEqualToValue={(option, value) => value.profileName === option.profileName}
+              renderInput={(params: any) => (
+                <TextField
+                  {...params}
+                  error={!!errors.defaultProfileId}
+                  helperText={errors.defaultProfileId ? errors.defaultProfileId.message : ''}
+                  placeholder="Default billing profile"
+                />
+              )}
+              getOptionLabel={(option: BillingProfileModel) => option.profileName || ''}
```

Comment:
I agree with displaying a billing profile by its human-readable name, but what do you think about displaying the UUID secondarily?  Maybe something like
```
name (aaaa-bbbb-cccc-dddd)
```
where the UUID is styled to grey.  I'm thinking in case the user wants to interact with our billing profile endpoints, they'll need the UUID.

Also not sure if it's worth calling out in some way at this stage that even if a billing profile is accessible to the user, they will also need access to its underlying billing account… maybe the dataset creation error message in that case covers our bases well enough.